controllers/memcached/memcached_controller.go

```diff
@@ -182,7 +180,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 
 	instance.Status.Conditions.Init(&cl)
 	instance.Status.ObservedGeneration = instance.Generation
-	instance.Status.MTLSCert = ""
+
+	if instance.Spec.TLS.MTLS.SslVerifyMode == "Request" || instance.Spec.TLS.MTLS.SslVerifyMode == "Require" {
+		// MTLS cert secrets
+		cl = append(cl, *condition.UnknownCondition(memcachedv1.MTLSInputReadyCondition, condition.InitReason, condition.InputReadyInitMessage))
+		instance.Status.MTLSCert = ""
```

Comment:
now that I think of it again, I am not sure if we should reset it on each iteration. if we fail in the middle for some other reason, the name of the cert name of the previous run would be still valid, right?